src/client/components/pages/Courses/MeetingTimesList.tsx

```diff
@@ -0,0 +1,472 @@
+    }
+  });
+
+  return (
+    <div className="meeting-times-section">
+      <ul>
+        {currentMeetings.map(
+          (meeting) => (
+            <StyledMeetingRow
+              key={meeting.id}
+              isRowExpanded={meeting.id === currentMeetingId}
+            >
+              <StyledDeleteButton>
+                <BorderlessButton
+                  id={`deleteButton${meeting.id}`}
+                  variant={VARIANT.DANGER}
+                  onClick={
+                    (): void => {}
+                  }
+                >
+                  <FontAwesomeIcon icon={faTrash} />
+                </BorderlessButton>
+              </StyledDeleteButton>
+              {
+                meeting.id === currentMeetingId
+                  ? (
+                    <>
+                      <StyledDay>
+                        <Dropdown
+                          id="meetingDay"
+                          name="meetingDay"
+                          label="Meeting Day"
+                          options={[{ value: '', label: '' }]
+                            .concat(days.map((day) => ({
+                              value: day,
+                              label: dayEnumToString(day),
+                            })))}
+                          value={currentDay}
+                          onChange={(event
+                          : React.ChangeEvent<HTMLSelectElement>): void => {
+                            setCurrentDay(event.currentTarget.value as DAY);
+                            setCurrentMeetings(currentMeetings.map(
+                              (currentMeeting) => (
+                                currentMeeting.id === currentMeetingId
+                                  ? {
+                                    ...currentMeeting,
+                                    day: event.currentTarget.value as DAY,
+                                  }
+                                  : currentMeeting
+                              )
+                            ));
+                          }}
+                          hideError
+                          isRequired
+                          isLabelVisible={false}
+                        />
+                      </StyledDay>
+                      <StyledTimeslot>
+                        <ButtonDropdownMenu
+                          alt="Timeslot Button"
+                          label={<FontAwesomeIcon icon={faAngleDown} size="sm" />}
+                          variant={VARIANT.BASE}
+                        >
+                          {meetingTimeSlots.map(({
+                            label,
+                            start,
+                            end,
+                          }) => (
+                            <ButtonDropdownMenuItem
+                              onClick={() => {
+                                const startTime = convert12To24HourTime(start);
+                                const endTime = convert12To24HourTime(end);
```

Comment:
Ah good catch! I think I was in the mode of trying to make sure everything was in 24 hour time so that the values would populate correctly in the start and end time fields, but this conversion was not necessary. This was updated in cbd411e.